### ISSUE TITLE
Allow for different date formats

### ISSUE THREAD

**Issue by jamesozzie** [google/site-kit-wp#1518]
## Feature Description

As per the below WordPress support forum topic one possible enhancement would be to allow different date formats to be used within the various dashboards.
https://wordpress.org/support/topic/sitekit-doesnt-display-stats-correctly/#post-12795818

![image](https://user-images.githubusercontent.com/41326532/81291760-51b17300-9062-11ea-88f4-a7ab0dfbd884.png)

Some users may prefer to have date periods in the DD/MM/YYYY or other format. While trickier to achieve tied to their chosen format using their WordPress general settings. 

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## QA Brief

* 

## Changelog entry

*

**Comment by ronnyadsetts** [comment]
Yeah, US date formats are pretty illogical and hard to parse for everyone else in the world. Please follow the site locality settings if you're sticking with a full date on these graphs.
